modules/apps/commerce/commerce-channel-web/src/main/java/com/liferay/commerce/channel/web/internal/display/context/CommerceChannelDisplayContext.java

```diff
@@ -368,6 +376,17 @@ public PortletURL getPortletURL() throws PortalException {
 		return portletURL;
 	}
 
+	public List<AccountEntry> getSupplierAccountEntries()
+		throws PortalException {
+
+		return _accountEntryService.getAccountEntries(
```

Comment:
as said before, use the search instead of adding this new method

Reply:
Done